player/public/coreplugins/org.ekstep.repo-1.0/renderer/plugin.js

```diff
@@ -90,12 +90,12 @@ Plugin.extend({
      * @return {Boolean}       [If the path is already present then it will return true else false in the callback]
      */
     isAvailable: function(repo, cb) {
-        var isAvailable = undefined;
         if (org.ekstep.pluginframework.resourceManager.registeredRepos) {
-            isAvailable = _.find(org.ekstep.pluginframework.resourceManager.registeredRepos, function(r) {
-                return r.basePath == repo;
+            var repos = _.filter(org.ekstep.pluginframework.resourceManager.registeredRepos, function(r) {
+                return r.basePath !== repo
             });
-            isAvailable ? cb(true) : cb(false);
+            org.ekstep.pluginframework.resourceManager.registeredRepos = repos;
+            cb(false);
```

Comment:
why cb(false) here @akashgupta9990  How it works?

Reply:
Previous logic was 
if  repo is not available call callback with param as false.
If repo is available then call callback with param as true.

Now Logic is
If repo is not available call callback with param as false.
if repo is available, remove that repo and call callback with param as false.

Callback with param true was there to not registered repo again.
But now we are removing repo if it is available and adding it again with index as 0